product_supplierinfo_for_customer/__openerp__.py

```diff
@@ -17,20 +17,24 @@
 ##############################################################################
 {
     "name": "Use product supplier info for customers too",
-    "version": "8.0.1.0.0",
-    "author": "OdooMRP team,"
-              "AvanzOSC,"
-              "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
+    "version": "8.0.2.0.0",
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Vauxoo, "
               "Odoo Community Association (OCA)",
     "website": "http://www.odoomrp.com",
     "category": "Sales Management",
     "license": 'AGPL-3',
     "depends": [
-        "base",
+        "sale",
```

Comment:
You shouldn't add dependencies to current module. Create new modules `product_supplierinfo_for_customer_sale` and ...purchase with that purpose, or use new module https://github.com/OCA/server-tools/pull/876 for conditional dependencies